components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -58,6 +58,7 @@ FORWARD_DECLARE_TEST(BraveSyncServiceTest, ProcessOtherBookmarksFolder);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, ProcessOtherBookmarksChildren);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, CheckOtherBookmarkRecord);
 FORWARD_DECLARE_TEST(BraveSyncServiceTest, CheckOtherBookmarkChildRecord);
+FORWARD_DECLARE_TEST(BraveSyncServiceTest, AddNonClonedBookmarkKeys);
```

Comment:
Why do we need this?  the test only accesses `AddNonClonedBookmarkKeys` which is a public static method